src/app/collection/pages/tags/tag-with-cyber/components/header/header.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input } from '@angular/core';
+import { Router } from '@angular/router';
+import { TagsService } from 'app/core/learning-object-module/tags/tags.service';
+@Component({
+  selector: 'clark-with-cyber-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+export class HeaderWithCyberComponent {
+
+  @Input() collectionAbv: string;
```

Comment:
nit, since this component is for the header of just the with-cyber component, can we name this input value to be tagName, or something instead of collectionAbv since that's not really what it will be?